reference/markdown/Format-comparison.md

```diff
@@ -0,0 +1,34 @@
+Although the original [Markdown DITA](./Markdown-syntax-reference.md) format and the [MDITA](./MDITA-syntax-reference.md) format for _LwDITA_ share some common syntax, there are several differences to consider when choosing which format to use.
+
+<!-- Re-use short descriptions from syntax topics -->
+
+- In 2015, the original _DITA-OT Markdown_ plug-in introduced a series of conventions to convert [Markdown] content to DITA, and vice-versa. This Markdown flavor was called _“Markdown DITA”_. The `markdown` format adds several complementary constructs to represent DITA content in Markdown, beyond those proposed for the [MDITA](./MDITA-syntax-reference.md) format in the [Lightweight DITA][LwDITA] specification drafts.
+
+- In 2017, the Markdown plug-in was superseded by the _LwDITA_ plug-in, which was bundled with DITA-OT 3.0, and added new formats for [Lightweight DITA][LwDITA]. The `mdita` format implements the subset of Markdown features proposed in the latest specification drafts, but differs in some ways from the original [Markdown DITA](./Markdown-syntax-reference.md) format.
+
+The following table provides an overview of differences between the `markdown` and `mdita` formats.
+
+| Features                 | Markdown DITA                   | MDITA (LwDITA)             |
+| ------------------------ | ------------------------------- | -------------------------- |
+| Format                   | `markdown`                      | `mdita`                    |
+| LwDITA                   | –                               | ✔                          |
+| First ¶                  | Body ¶                          | Short description          |
+| Subeadings               | Nested topics                   | Sections                   |
+| Topic IDs                | Attributes or title             | Generated from title       |
+| Output class             | Defined via attributes          | –                          |
+| Profiling atts           | Defined via attributes          | –                          |
+| Topic types              | Defined via attributes          | –                          |
+| Schemas                  | YAML frontmatter                | –                          |
+| Tables                   | CALS                            | Simpletable                |
```

Comment:
CALS table model is the old name. DITA uses OASIS exchange table model.

Reply:
'CALS table' means something to me. It's a term with which I'm familiar. I had to look up 'OASIS exchange table model.' Perhaps it's worth adding a footnote that 'OASIS exchange table model' is a subset of CALS or a [link](https://www.oasis-open.org/specs/tm9901.html)?

Reply:
@jelovirt Are file name extensions really used for format recognition?

[Usage](https://github.com/jelovirt/org.lwdita/wiki/Usage) says this:

> The `@format` attribute value must be set to `mdita`, `markdown`, or `md` in order to recognize files as Markdown DITA; The `@format` attribute value must be set to `hdita` in order to recognize files as HTML DITA. The file extension is not used to recognize format.

Reply:
@infotexture Sorry for the delayed response. Yes, file name extensions are used when there is no other way to recognize what the format is. For example, when the input file is `topic.md`, the file name extension is used to look up which parser to use to parse `md` files.

So that last sentence is not longer correct, because DITA-OT does use it when `@format` information is not available.